.github/workflows/ci.yaml

```
@@ -0,0 +1,32 @@
+          submodules: recursive
+      - name: Install Kontrol
+        run: |
+          bash <(curl https://kframework.org/install)
+          kup install kontrol
+      - name: Build and Run Proofs
+        run: |
+          kontrol build
+          kontrol prove --config-profile setup
+          kontrol prove --config-profile tests
+      - name: View Results
+        run: kontrol list
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)